Celeste.Mod.mm/Mod/Registry/DecalRegistry.cs

```diff
@@ -12,9 +13,15 @@ namespace Celeste.Mod {
     /// Allows custom decals to have properties that are otherwise hardcoded, such as reflections, parallax, etc.
     /// </summary>
     public static class DecalRegistry {
+        /// <summary>
+        /// Mapping of decal paths to decal registry properties.
+        /// </summary>
+        public static readonly Dictionary<string, DecalInfo> RegisteredDecals = new Dictionary<string, DecalInfo>();
 
-        // string is propertyName
-        public static Dictionary<string, Action<Decal, XmlAttributeCollection>> PropertyHandlers = new Dictionary<string, Action<Decal, XmlAttributeCollection>>() {
+        /// <summary>
+        /// Everest-defined decal registry properties. See <see cref="AddPropertyHandler"/> to add a custom property.
+        /// </summary>
+        internal static Dictionary<string, Action<Decal, XmlAttributeCollection>> PropertyHandlers = new Dictionary<string, Action<Decal, XmlAttributeCollection>>() {
```

Comment:
can we check if any mods relying on this being `public`?

Reply:
I did a GitHub wide search for all of the interface changes + tested it with SJ which is about as thorough as I can get haha